internal/pod_models.go

```diff
@@ -479,6 +494,10 @@ func GetPodSpec(cluster *fdbv1beta2.FoundationDBCluster, processClass fdbv1beta2
 		if processClass == fdbv1beta2.ProcessClassStorage && cluster.GetStorageServersPerPod() > 1 {
 			sidecarContainer.Env = append(sidecarContainer.Env, corev1.EnvVar{Name: "STORAGE_SERVERS_PER_POD", Value: fmt.Sprintf("%d", cluster.GetStorageServersPerPod())})
 		}
+
+		if processClass == fdbv1beta2.ProcessClassLog && cluster.GetStorageServersPerPod() > 1 {
+			sidecarContainer.Env = append(sidecarContainer.Env, corev1.EnvVar{Name: "LOG_SERVERS_PER_POD", Value: fmt.Sprintf("%d", cluster.GetStorageServersPerPod())})
```

Comment:
```suggestion
		if processClass == fdbv1beta2.ProcessClassLog && cluster.GetLogServersPerPod() > 1 {
			sidecarContainer.Env = append(sidecarContainer.Env, corev1.EnvVar{Name: "LOG_SERVERS_PER_POD", Value: fmt.Sprintf("%d", cluster.GetLogServersPerPod())})
```

Please make sure that the unit tests are passing. This should make sure that we catch such issues.